main/cicd/cicd-pipeline/serverless.yml

```diff
@@ -15,7 +15,6 @@ provider:
   # All other ${* references will be resolved by CloudFormation
   # See https://forum.serverless.com/t/getting-handle-accountid-in-serverless-config/946/11 and
   # See https://github.com/serverless/serverless/issues/5011
-  variableSyntax: '\$\{((((self|opt|deep|cf):)|file)((?!\$\{).)+?)}'
```

Comment:
(I've left this comment elsewhere, feel free to respond on any one of them and resolve the rest) This seem important. Why do we remove it? If we really don't need it, then maybe remove the preceding comments